containers/kubernetes/how-to/enable-disable-ssh.mdx

```diff
@@ -0,0 +1,65 @@
+---
+meta:
+  title: How to enable or disable SSH
+  description: This page explains how to enable or disable the SSH service in your nodes
+content:
+  h1: How to enable or disable SSH
+  paragraph: This page explains how to enable or disable the SSH service in your nodes
+tags: kubernetes kapsule-cluster
+dates:
+  validation: 2023-02-23
+  posted: 2023-02-23
+categories:
+  - containers
+---
+
+By default, an SSH server is installed and configured on the nodes of your Kubernetes Kapsule cluster.
+To improve the security of your Kapsule cluster, you can disable this SSH server on your nodes.
+
+You can manage and edit the status of the SSH server of your nodes from the [Scaleway Console](https://console.scaleway.com) or by using the [API](https://developers.scaleway.com/en/).
+Here, we show you how to do it from the console.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
+  - This feature is available on the nodes of your cluster
+</Message>
+
+## How to know if this feature is available on the nodes of your cluster
+
+1. Open a terminal program.
+2. Enter the command below into the terminal. Make sure you replace `path_to_your_kubeconfig` with the path to your kubeconfig file.
+    ```bash
+    KUBECONFIG=path_to_your_kubeconfig kubectl get nodes -o custom-columns=NAME:.metadata.name,"AGENT-VERSION":".metadata.annotations.k8s\.scaleway\.com/agent-version"
+    ```
+
+    <Message type="tip">
+      To find the kubeconfig of your Kapsule cluster, log into the Scaleway console and click **Kubernetes** in the **Containers** section of the side menu.
+      A list of your Kubernetes clusters will display, click on the one you want to verify. In the Download kubeconfig section at the bottom of the page, click on **Download file** to download the kubeconfig.
+    </Message>
+3. This feature is available on all nodes of your cluster if `AGENT-VERSION` is at least v1.0.0 for all your nodes.
+    ```
+    NAME                                             AGENT-VERSION
+    scw-k8s-silly-cartwright-default-ca35b74a4b644   v1.0.0
+    scw-k8s-silly-cartwright-default-ca978112ca1bb   v1.0.0
+    ```
+    <Message type="tip">
+      If this feature is not available on a specific node of your cluster, try to replace it.
+    </Message>
+
+## How to enable or disable SSH on the nodes of a pool
+
+1. Click **Kubernetes** in the **Containers** section of the [Scaleway Console](https://console.scaleway.com) side menu
```

Comment:
Hi @Tomy2e, 
can you add a trailing dot at the end of your sentences in the steps, please?